suprb/suprb.py

```diff
@@ -147,15 +147,27 @@ def fit(self, X: np.ndarray, y: np.ndarray, cleanup=False):
 
         # Fill population before first step
         if self.n_initial_rules > 0:
-            self._discover_rules(X, y, self.n_initial_rules)
+            try:
+                self._discover_rules(X, y, self.n_initial_rules)
+            except:
+                warnings.warn("Error in rule discovery first iteration")
```

Comment:
Change to:
"An error has occured when trying to discover rules for the first time. This is likely due to a bad configuration".

Additionally, please include the original error message